modules/osd-create-cluster-ccs-aws.adoc

```diff
@@ -0,0 +1,227 @@
+. If you opted to configure a cluster-wide proxy, provide your proxy configuration details on the *Cluster-wide proxy* page:
++
+.. Enter a value in at least one of the following fields:
+** Specify a valid *HTTP proxy URL*.
+** Specify a valid *HTTPS proxy URL*.
+** In the *Additional trust bundle* field, provide a PEM encoded X.509 certificate bundle. The bundle is added to the trusted certificate store for the cluster nodes. An additional trust bundle file is required if you use a TLS-inspecting proxy unless the identity certificate for the proxy is signed by an authority from the {op-system-first} trust bundle. This requirement applies regardless of whether the proxy is transparent or requires explicit configuration using the `http-proxy` and `https-proxy` arguments.
++
+.. Click *Next*.
++
+For more information about configuring a proxy with {product-title}, see _Configuring a cluster-wide proxy_.
+
+. In the *CIDR ranges* dialog, configure custom classless inter-domain routing (CIDR) ranges or use the defaults that are provided.
++
+[NOTE]
+====
+If you are installing into a VPC, the *Machine CIDR* range must match the VPC subnets.
+====
++
+[IMPORTANT]
+====
+CIDR configurations cannot be changed later. Confirm your selections with your network administrator before proceeding.
+====
++
+. Accept the default application ingress settings, or to create your own custom settings, select *Custom Settings*.
+
+.. Optional: Provide route selector.
+.. Optional: Provide excluded namespaces.
+.. Select a namespace ownership policy.
+.. Select a wildcard policy.
++
+For more information about custom application ingress settings, click the information icon provided for each setting.
+
+. On the *Cluster update strategy* page, configure your update preferences:
+.. Choose a cluster update method:
+** Select *Individual updates* if you want to schedule each update individually. This is the default option.
+** Select *Recurring updates* to update your cluster on your preferred day and start time, when updates are available.
++
+[NOTE]
+====
+You can review the end-of-life dates in the update lifecycle documentation for {product-title}. For more information, see link:https://access.redhat.com/documentation/en-us/openshift_dedicated/4/html/introduction_to_openshift_dedicated/policies-and-service-definition#osd-life-cycle[OpenShift Dedicated update life cycle].
+====
++
+.. Provide administrator approval based on your cluster update method:
+** Individual updates: If you select an update version that requires approval, provide an administrator’s acknowledgment and click *Approve and continue*.
+** Recurring updates: If you selected recurring updates for your cluster, provide an administrator’s acknowledgment and click *Approve and continue*. {cluster-manager} does not start scheduled y-stream updates for minor versions without receiving an administrator’s acknowledgment.
```

Comment:
I don't think this could be done from wizard at the time of installation. The administrator approval could be done once the cluster is ready and user want to update to the next available versions that required approval. I would suggest to remove the section.